js/src/common/Application.tsx

```diff
@@ -247,36 +290,34 @@ export default class Application {
 
   /**
    * Determine the current screen mode, based on our media queries.
-   *
-   * @returns {String} - one of "phone", "tablet", "desktop" or "desktop-hd"
    */
-  screen() {
+  screen(): 'phone' | 'tablet' | 'desktop' | 'desktop-hd' {
```

Comment:
Maybe this one should be an exported type alias, since it's used in a lot of external code?